src/lib/components/CapturePool.svelte

```diff
@@ -3,17 +3,39 @@
   import type { Color } from "$lib/type";
   import ChessPiece from "./ChessPiece.svelte";
 
-
   export let color: Color;
-  export let capturedPieces: Piece[] | null[];
+  export let capturedPieces: Piece[];
+
+  $: sortedCapturedPieces =
+    capturedPieces.length > 1
+      ? capturedPieces.sort((a, b) => {
+          return a?.value - b?.value;
+        })
+      : capturedPieces;
+
+  $: playersCapturedValue = capturedPieces.reduce((acc, piece) => {
+    acc = piece?.value! + acc;
+    return acc;
+  }, 0);
+  $: opponentsCapturedValue = capturedPieces[0]
```

Comment:
Does not need to be a turnary because reduce sets initial value to 0 and will return 0 if array is length 0